pkgs/tools/typesetting/rubber/default.nix

```diff
@@ -1,23 +1,34 @@
 { fetchurl, stdenv, python2Packages, texinfo }:
 
-stdenv.mkDerivation rec {
-  name = "rubber-1.3";
+let
+  pypkgs = python2Packages;
```

Comment:
The idea was to avoid having to specify ```python2Packages``` 3 times. If we need to move to python 3 at some point, that means changing only one variable.